core/math/rect2.cpp

```diff
@@ -95,26 +95,31 @@ bool Rect2::intersects_segment(const Point2 &p_from, const Point2 &p_to, Point2
 
 bool Rect2::intersects_transformed(const Transform2D &p_xform, const Rect2 &p_rect) const {
 
-	//SAT intersection between local and transformed rect2
-
-	Vector2 xf_points[4] = {
+	Vector2 p_xf_points[4] = {
```

Comment:
Well, usually we only prepend parameters with `p_`. That's kind of confusing if you prepend a variable too.

Reply:
Oops. A case of uncritical copy-paste. Fixed!

Reply:
That's what I suspected. ^^